templates/pioneer-study/esper-study-telemetry.1.parquetmr.txt

```diff
@@ -0,0 +1,42 @@
+message esper-study-telemetry {
+  required group metadata {
+    required int64  Timestamp;
+    required binary documentId (UTF8);
+    required binary pioneerId (UTF8);
+    required binary studyName (UTF8);
+    optional binary geoCity (UTF8);
+    optional binary geoCountry (UTF8);
+    optional binary normalizedChannel (UTF8);
+  }
+  required binary defaultSearchEngine (UTF8);
+  required binary locale (UTF8);
+  required binary os (UTF8);
+  required binary normalizedChannel (UTF8);
```

Comment:
Isn't this already in the metadata?

Reply:
Yes, but it is supposed to be in both the metadata and the payload according to the phd.